src/pages/model/stackview/ModelPageStackViewContent.svelte

```diff
@@ -30,24 +30,27 @@
 <h1 class="sr-only">{$t('content.index.toolProcessCards.model.title')}</h1>
 <div class="flex flex-col h-full bg-backgrounddark">
   <div class="flex justify-center space-x-10 py-5 text-xl border-b-3 border-b-gray-200">
-    <Information
-      underlineIconText={false}
-      isLightTheme={false}
-      iconText={$t('content.model.output.estimatedGesture.iconTitle')}
-      titleText={$t('content.model.output.estimatedGesture.descriptionTitle')}
-      bodyText={$t('content.model.output.estimatedGesture.descriptionBody')} />
-    {#if $bestPrediction && $bestPrediction.confidence.isConfident}
-      <p class="font-semibold text-2xl">
-        {$bestPrediction.name}
-      </p>
-      <p class="bg-secondary text-white rounded w-15 text-center">
-        {Math.round($bestPrediction.confidence.currentConfidence * 100)}%
-      </p>
-    {:else}
+    <div class="flex space-x-2 justify-center">
+      <Information
+        iconTextClass="font-semibold text-2xl"
+        iconText={$t('content.model.output.estimatedGesture.iconTitle')}
+        underlineIconText={false}
+        isLightTheme={false}
+        titleText={$t('content.model.output.estimatedGesture.descriptionTitle')}
+        bodyText={$t('content.model.output.estimatedGesture.descriptionBody')} />
+    </div>
+    <div class="flex space-x-10 min-w-340px">
```

Comment:
I've provided some suggestions that'll keep everything centred while allowing the right side to grow. See what you think.

Reply:
I think it's pretty good in that it truly ensures that the estimated action is not going to move. A downside is that the worst looking scenario would be like below:
<img width="1038" alt="Screenshot 2024-01-12 at 15 11 09" src="https://github.com/microbit-foundation/ml-trainer/assets/145345672/79560bd3-57ee-434d-ac8d-4b0a5553494b">

I think the trade off is either having estimated action shifting with long name VS not having the estimated action centred. What do you think? Which one is better?

Reply:
Worse on narrower screens as well. From an accessibility point of view the shift is bad. Maybe having everything aligned left rather than center would be better. Could discuss at the stand up.

Reply:
I can't tell whether I am just not used to it, or it looks weird. Anyway a preview of what it might look like for discussion
<img width="960" alt="Screenshot 2024-01-12 at 16 28 14" src="https://github.com/microbit-foundation/ml-trainer/assets/145345672/51ebe95f-d33f-4476-b2e5-73acd1d3bc2b">